armi/tests/test_fuelHandlers.py

```diff
@@ -499,11 +498,20 @@ def test_buReducingAssemblyRotation(self):
     def test_buildRingSchedule(self):
         fh = fuelHandlers.FuelHandler(self.o)
         schedule, widths = fh.buildRingSchedule(17, 1, jumpRingFrom=14)
+
+        # test 1
         self.assertEqual(
             schedule, [13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 14, 15, 16, 17]
         )
         self.assertEqual(widths, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
 
+        # test 2
+        schedule, widths = fh.buildRingSchedule(1, 17, jumpRingFrom=0)
+        self.assertEqual(
+            schedule, [17, 16, 15, 14, 13, 12, 11, 10, 2, 3, 4, 5, 6, 7, 8, 9, 1]
```

Comment:
This pattern confuses me. I'm not sure why the ring scheduler would produce this output from that input in the buildRingSchedule method. I think this might be a bug with the jumpRingFrom equal to 0?

Reply:
might be a bug. weird that it jumps at ring 9. Probably worth digging into. 
Or making a ticket about that and changing jumpRingFrom here to be like `5`.

Reply:
That's interesting.  I just picked some arbitrary numbers for the test, of course. Well, it looks like that bore fruit really fast. Tests work!

The issue definitely arises there though: 

https://github.com/terrapower/armi/blob/b94744fdf12f247828dd07bf99e251651b494a13/armi/physics/fuelCycle/fuelHandlers.py#L1186-L1190

I assumed this was meant to catch "jumpFromRing = 0", but is this perhaps a mistake and this was meant to test for "jumpFromRing is None"?

Reply:
Ticket opened:  https://github.com/terrapower/armi/issues/498

Writing unit tests works!